entries/QUnit.config.xml

```diff
@@ -35,6 +35,11 @@
 				<p>Prior to version 1.16.0, this property was called <code>module</code>.</p>
 			</desc>
 		</property>
+		<property name="moduleId" type="Array" default="undefined">
+			<desc>
+				This property allows QUnit to run specific modules by a hashed string using their module name. You can specify one or multiple modules to run.
```

Comment:
Fixed, I've borrowed the wording to clarify the description for `testId` as well.